plextraktsync/sync.py

```diff
@@ -157,9 +157,7 @@ def sync_watched(self, m: Media, dry_run=False):
         if m.watched_on_plex:
             if not self.config.plex_to_trakt["watched_status"]:
                 return
-            logger.info(f"Marking as watched in Trakt: {m}")
```

Comment:
dry_run should still print what it will do, that's the point of dry run to see what would happen

also seems too much logic was moved to media class that should be in sync class (which has the dry-run flag)

Reply:
refactor attempt:
- https://github.com/Taxel/PlexTraktSync/pull/1140

the logic should stay pretty much the same.